### DISCUSSION TITLE
shortcut Key  doesn't Work  On Jetbrains

### DISCUSSION THREAD

**Discussion by MOHAMMADmiZAN** [community/community#10904] [Copilot Conversations] [answered]
{alt+],alt+[ } Shortcut Key  doesn't Work  On jetbrains Ide {WebStorm,PhpStorm}

**Comment by jansorg** [comment, marked as answer]
@MOHAMMADmiZAN There are a few problems with the shortcut on non-English keyboard layouts at this time. As a workaround, you can reconfigure the shortcut at `Preferences > Keymap > Plugins > GitHub Copilot`. Most likely we'll add additional shortcuts which are more suitable for non-English layouts.

**Comment by bjackson** [comment]
I'm on a US Mac keyboard and the shortcut doesn't work for me either.
